### PR TITLE
Sample app update to use the current config.

### PR DESCRIPTION
**Pull request by ts-leojoseph-r**
# Description
- Update sample app to use current config.
- Updated documentation.

# Checklist
- [x] I have read the [contributing guidelines](CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `fastlane ci` without errors

### CODE REVIEW

File: USERGUIDE.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ All the MiniApp files downloaded by the MiniApp iOS library are cached locally
 
 ## Requirements
 
-This module supports **iOS 11.0 and above**. It has been tested on iOS 11.0 and above.
+This module supports **iOS 13.0 and above**. It has been tested on iOS 13.0 and above.
```
Comment (Climbatize) [inline USERGUIDE.md:19]: this change is already waiting to be pushed on candidate branch

File: Example/Controllers/ViewController.swift [hunk 2]
```diff
@@ -103,10 +103,10 @@ class ViewController: UIViewController {
     }
 
     func compareMiniAppMetaData(miniAppInfo: MiniAppInfo, completionHandler: @escaping (Result<Bool, Error>) -> Void) {
-        guard let downloadedManifest = MiniApp.shared().getDownloadedManifest(miniAppId: miniAppInfo.id) else {
+        guard let downloadedManifest = MiniApp.shared(with: Config.getCurrent()).getDownloadedManifest(miniAppId: miniAppInfo.id) else {
             return completionHandler(.success(true))
         }
-        MiniApp.shared().getMiniAppManifest(miniAppId: miniAppInfo.id, miniAppVersion: miniAppInfo.version.versionId) { (result) in
+        MiniApp.shared(with: Config.getCurrent()).getMiniAppManifest(miniAppId: miniAppInfo.id, miniAppVersion: miniAppInfo.version.versionId) { (result) in
```
Comment (Climbatize) [inline Example/Controllers/ViewController.swift:109]: do you think we should rename `getCurrent` to just `current`?

**Review by Climbatize — APPROVED** [review]
LGTM, just 2 comments

File: USERGUIDE.md (hunk 1 above)
Comment (ts-leojoseph-r) [reply USERGUIDE.md:19]: Oh ok. So you don't want me to push this `USERGUIDE` change in `candidate` branch ? I'm going to push the `ViewController` changes anyway, let me knowl

File: Example/Controllers/ViewController.swift (hunk 2 above)
Comment (ts-leojoseph-r) [reply Example/Controllers/ViewController.swift:109]: I am thinking of `getCurrentConfig`  or may be `currentConfig` 🤔 . But I don't want this refactor to be pushed before the release. But I will add this in To-do

**Review by corycaywood — APPROVED** [review]
LGTM 👍

File: USERGUIDE.md (hunk 1 above)
Comment (Climbatize) [reply USERGUIDE.md:19]: that's fine, not a big deal just merge it. I can manage the change on my side

**Pull request merged**